### DISCUSSION TITLE
Prisma Session Store

### DISCUSSION THREAD

**Discussion by aresthegofwar** [prisma/prisma#18379] [Q&A]
is there any prisma session store for cookies?

**Comment by ludralph** [comment]
Hi @aresthegofwar 👋 

Within the ORM, Prisma doesn't provide this functionality however, there is a third-party library that provides this. You can take a look [here](https://github.com/kleydon/prisma-session-store).

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by ludralph** [reply]
Hi @aresthegofwar 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by ludralph** [reply]
Hi @aresthegofwar

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly mark this Discussion as answered to indicate that your issue has been resolved.

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
